src/SignalR/clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -40,7 +41,7 @@
     "src/**/*"
   ],
   "peerDependencies": {
-    "@aspnet/signalr": "^1.0.0-preview3"
+    "@aspnet/signalr": "link:../signalr"
```

Comment:
Does this get resolved in the `package.json` that ends up in the final `.tgz` package?